src/Raygun.Blazor/RaygunBrowserInterop.cs

```diff
@@ -216,10 +220,8 @@ public async ValueTask DisposeAsync()
                 await RaygunScriptReference.DisposeAsync();
             }
 
-            if (_dotNetReference is not null)
-            {
-                _dotNetReference.Dispose();
-            }
+            _dotNetReference.Dispose();
```

Comment:
I wouldn't recommend removing the null check here. If someone doesn't want to report data from Javascript files, this could easily be null, and then you'll crash the app as something is trying to be disposed.